wger/settings_global.py

```diff
@@ -407,3 +395,18 @@
     'EMAIL_FROM': 'wger Workout Manager <wger@example.com>',
     'TWITTER': False
 }
+
+# facebook, twitter and google API keys
```

Comment:
For clarity... any sensitive information that our app needs should not be made public. While changing this please also use env variables for the secret key